JS/main.js

```diff
@@ -19,4 +20,120 @@ links.forEach((link) => link.addEventListener('click', () => {
+  {
+    reverseCard:'kwacha',
+    cardName:'Multi-Post Stories',
+    mainText: `A daily selection of privately personalized reads; no accounts or
+    sign-ups required. has been the industry's standard dummy text
+    ever since the 1500s, when an unknown printer took a standard
+    dummy text.`,
+    cardLang: languages('css','html','bootstrap','ruby'),
+    image: `./placeholder.png`,
+
+  },
+  {
+    reverseCard:'reverse-row',
+    cardName:'Multi-Post Stories',
+    mainText: `A daily selection of privately personalized reads; no accounts or
+    sign-ups required. has been the industry's standard dummy text
+    ever since the 1500s, when an unknown printer took a standard
+    dummy tex.`,
+    cardLang: languages('css','html','bootstrap','ruby'),
+    image: `./placeholder.png`,
+    
+  },
+  {
+    reverseCard:'kwacha',
+    cardName:'Multi-Post Stories',
+    mainText: `A daily selection of privately personalized reads; no accounts or
+    sign-ups required. has been the industry's standard dummy text
+    ever since the 1500s, when an unknown printer took a standard
+    dummy text.`,
+    cardLang: languages('css','html','bootstrap','ruby'),
+    image: `./placeholder.png`,
+    
+  },
+  {
+    reverseCard:'reverse-row',
+    cardName:'Multi-Post Stories',
+    mainText: `A daily selection of privately personalized reads; no accounts or
+    sign-ups required. has been the industry's standard dummy text
+    ever since the 1500s, when an unknown printer took a standard
+    dummy text.`,
+    cardLang: languages('css','html','bootstrap','ruby'),
+    image: `./placeholder.png`,
+    
+  }
+
+]
+
+function popup() {
+  boxDiv.classList.add('active')
+}
+
+function createCard(cardData){
+  cardcontainer.innerHTML = ` `;
+  cardData.forEach((card, pos) => {
+    const cardMaster = `
+    <li class="card-sec ${card.reverseCard}">
+          <div class="placeholder1">
+            <img
+              class="image1"
+              src="${card.image}"
+              alt="image placeholder"
+            />
+          </div>
+          <div class="card1">
+            <h2 class="post">${card.cardName}</h2>
+            <p class="text">
+              ${card.mainText}
+            </p>
+            <ul class="lang">
+              ${card.cardLang}  
+            </ul>
+            <button class="button pop-${pos}"  type="button">See Project</button>
+          </div>
+        </li>`
+        cardcontainer.insertAdjacentHTML('afterbegin', cardMaster);
+  })
+ 
+} 
+
+
+createCard(cardsinfo)
+
+const modal = document.querySelector('.pop-up--overlay');
+const openbtn = document.querySelectorAll('.button');
+console.log(openbtn);
+const closebtn = document.querySelector('.close-btn');
+//listen for click
+
+//display popup
+function openModal(){
+  boxDiv.classList.add('active')
+}
+
+//close popup
+function closeModal(){
+  boxDiv.classList.remove('active')
+}
+
+openbtn.forEach((btn) => {
+  btn.addEventListener('click', openModal);
+})
```

Comment:
- [ ]  I noticed that every time I click on the see project button it only displays the information of the first popup instead of the one concerning it, kindly fix this and make sure each button displays the corresponding popup window

Reply:
My page have the similar information on all cards